README.rst

```diff
@@ -175,22 +162,8 @@ you can receive all the latest updates, and switch back to version 5.x behavior:
     >>> dt.tzinfo
     <DstTzInfo 'Europe/Vienna' CET+1:00:00 STD>
 
-Version 6 is on `branch master <https://github.com/collective/icalendar/>`_ with compatibility to Python versions ``3.7+`` and ``PyPy3``.
```

Comment:
In combination with the replacing the compatibility field with a version badge, removing this line eliminates all text of Python version compatibility. That's not good. See suggestion below.

Reply:
> It is compatible with Python versions 3.7 - 3.11, and PyPy3.

The above statement is in the README now.
If anything comes up, you can still add a PR.